src/signature_pad.js

```diff
@@ -404,7 +406,7 @@ SignaturePad.prototype._toSVG = function () {
 
         path.setAttribute('d', attr);
         path.setAttribute('stroke-width', (widths.end * 2.25).toFixed(3));
-        path.setAttribute('stroke', this.penColor);
+        path.setAttribute('stroke', color);
```

Comment:
Thanks! I've added a note in the README that calling `#toData` won't really work if signature was loaded using `#fromDataURL`.